apps/mi/base/kustomization.yaml

```diff
@@ -22,5 +22,4 @@ resources:
   - ../mi-staging-service/mi-staging-service-identity-binding.yaml
   - ../mi-staging-service/mi-staging-service-identity.yaml
   - ../mi-staging-service/mi-staging-service.yaml
-  - ../mi-sftp-server/mi-sftp-server.yaml
```

Comment:
doesn't this need adding in other places?

Reply:
Is it required in sbox? It's deployed to and running in test and stg via Flux V1, doesn't exist in prod.

Reply:
ok no worries